cats/cats-test/src/main/groovy/com/netflix/spinnaker/cats/cache/CacheSpec.groovy

```diff
@@ -95,7 +95,7 @@ abstract class CacheSpec extends Specification {
         '*TEST*'                | ['blaTEST', 'blaTESTbla', 'TESTbla']
         'bla*'                  | ['blaTEST', 'blaTESTbla', 'blaPest', 'blaFEST']
         'bla[TF]EST'            | ['blaTEST', 'blaFEST']
-        'bla?EST'               | ['blaTEST', 'blaFEST']
```

Comment:
This test previously passed because H2 treats regex statements as case sensitive by default. MySQL defaults to case insensitive, but that's preferable. Case insensitive search was a requested feature which cat-sql delivers.